mobile_config/src/settings.rs

```diff
@@ -15,8 +15,9 @@ pub struct Settings {
     /// Listen address. Required. Default to 0.0.0.0::8080
     #[serde(default = "default_listen_addr")]
     pub listen: SocketAddr,
-    /// File from which to load config server signing keypair
-    pub signing_keypair: String,
+    /// Base64 encoded string with bytes of helium keypair
+    #[serde(deserialize_with = "crate::deserialize_keypair")]
+    pub signing_keypair: Arc<Keypair>,
     /// B58 encoded public key of the default admin keypair
     pub admin_pubkey: String,
```

Comment:
Do we want to make this a `PubKey` like `mobile_config/src/client/settings.rs`?

Reply:
Yes that should be a `PubKey`